sources/packages/api/src/services/education-program/education-program.service.ts

```diff
@@ -79,6 +80,10 @@ export class EducationProgramService extends RecordDataModelService<EducationPro
       educationProgram.hasJointDesignatedInstitution;
     program.approvalStatus = educationProgram.approvalStatus;
     program.institution = { id: educationProgram.institutionId } as Institution;
+    if (educationProgram.programIntensity === "yes")
```

Comment:
Please avoid resolving it on API. I would recommend changing the values on Form.IO definition.